src/clocksi_interactive_tx_coord_fsm.erl

```diff
@@ -93,6 +96,15 @@
 %%% API
 %%%===================================================================
 
```

Comment:
The following changes add the possibility to start a static transaction that will be entirely handled by this coord.